backend/tests/integration/test_onboarding_api.py

```
@@ -0,0 +1,452 @@
+        """Test status for user with completed onboarding."""
+        user, onboarding = test_user_with_onboarding
+        
+        # Create auth headers for this user
+        from schemas.auth import LoginRequest
+        from services.auth_service import AuthService
+        import asyncio
+        
+        # This is a bit complex due to async nature, but necessary for integration test
+        # In a real app, you might have a helper function for this
+        
+        response = client.get("/api/v1/onboarding/status", headers={
```

Ruff (0.13.3)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)